nemo/export/utils.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import logging
+import os
+import pathlib
+import tarfile
+import tempfile
+import typing
+from pathlib import Path
+
+import numpy as np
+import torch
+import yaml
+
+log_format = "%(asctime)s %(name)s [%(levelname)s] %(message)s"
+logging.basicConfig(format=log_format)
+LOGGER = logging.getLogger("NeMo")
+
+# numpy doesn't know bfloat16, define abstract binary type instead
+np_bfloat16 = np.dtype('V2', metadata={"dtype": "bfloat16"})
+
+
+def unpack_nemo_ckpt(
```

Comment:
Import save restore connector t this level and use it. Also it should be called `unpack_nemo_file` - ptl ckpt is different from nemo file.

Reply:
Same reason above. This export to trt-llm needs to be self contained. Let me actually move this utils.py file into the trt-llm folder as well.